### ISSUE TITLE
Poor pagination in the Developer Guide

### ISSUE THREAD

**Issue by kiatkat** [kiatkat/pe#13]
![image.png](https://raw.githubusercontent.com/kiatkat/pe/main/files/de86f76e-773c-4b61-829d-d3ca89667a32.png)

![image.png](https://raw.githubusercontent.com/kiatkat/pe/main/files/b673d638-2dc9-468f-bb71-4c32dabf14a6.png)

![image.png](https://raw.githubusercontent.com/kiatkat/pe/main/files/e37560ee-6842-4180-8a23-e8771cbbc69e.png)

See attached examples. Similarly to the User Guide, the Developer Guide has isolated headers that can make it hard to follow for the reader.

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Page Breaks in the UG
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>Page breaks in the UG could've been improved for a smoother reading experience
>
>![Screenshot 2023-11-17 at 5.32.33 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S1/pe/master/files/c5988bc6-085b-49c8-bfa2-8b1e719a1536.png)
>
>
>![Screenshot 2023-11-17 at 5.31.33 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S1/pe/master/files/c2309963-3ebe-478e-8659-ddc5cb0ff2e5.png)
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2324S1/pe-interim#3976] [original labels: type.DocumentationBug severity.VeryLow]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>No details provided by team.

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
